task3/frontpage.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="frontpage.css"> 
+<head>
+   <!-- <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">-->
+    <title>NTW</title>
+</head>
+<body  >
+    <header class="blok "  >
+        
+        <div class="d1 box0"></div>
+        
+        <div class="d2 box1 ">
+        
+            <p class="avtor">NTW Сальдюн Андрей 2 курс 7 группа  asaldyun@gamil.com  v.0.0</p>
+            
+            <form action="sing in"> 
+                <button class="buttonsin" type="Sing in">Sing in</button>
+            </form>  
+        
+            <img class="prphoto" src="image/person-icon.png"
+            alt="profil photo">
+        
+            <form action="sub"> 
+                <button class="buttonsub" type="submit">Subscribe</button>
+            </form> 
+
+            <p class="prfinfo" > profill info</p>
+
+            <form >
+                <p><input type="date" placeholder="date" class="options" width="25%" ></p>
+                <p><input type="text" placeholder="name" class="options" width="25%" ></p>
+                <p><input type="text" placeholder="tag" class="options" ></p>
+                <p><input type="submit" value="filter" class="options" width="50px" ></p>
```

Comment:
In HTML5 it is preferable to use button type="submit" / type="reset" instead of input tags.
PS: input basically the biggest mistake in HTML standard cause it is mistake to try to mix "bulldogs and rhinoceros".